packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.html

```diff
@@ -0,0 +1,114 @@
+<div class="telecom-pack-xdsl-meetings">
+    <header class="widget-presentation-header">
+        <h2
+            class="widget-presentation-title"
+            data-translate="xdsl_meeting_slots"
+        ></h2>
+    </header>
+
+    <!-- LOADER -->
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <oui-message type="error" data-ng-if="$ctrl.errorMessage">
+        <span data-ng-bind-html="$ctrl.errorMessage"></span>
+    </oui-message>
+
+    <oui-message type="success" data-ng-if="$ctrl.successMessage">
+        <span data-ng-bind-html="$ctrl.successMessage"></span>
+    </oui-message>
+
+    <div
+        class="widget-presentation py-3 move-number-config ft-meeting"
+        data-ng-if="$ctrl.showMeetingSlots && !ctrl.loading && !$ctrl.successMessage"
+    >
+        <div class="row">
+            <div
+                class="font-weight-bold"
+                data-translate="xdsl_meeting_description"
+            ></div>
+
+            <div
+                data-ng-show="$ctrl.meetingSlots.canBookFakeMeeting && $ctrl.meetingSlots.slots.length"
+            >
+                <div class="input-group mt-3 col-md-12">
+                    <input
+                        type="radio"
+                        id="radioFakeMeeting"
+                        data-ng-model="$ctrl.select.meetingSlots.fakeMeeting"
+                        data-ng-disabled="!$ctrl.meetingSlots.canBookFakeMeeting"
+                        data-ng-value="true"
+                    />
+                    <label
+                        for="radioFakeMeeting"
+                        class="pl-2"
+                        data-translate="xdsl_meeting_fake_meeting"
+                    >
+                    </label>
+                </div>
+                <div class="input-group mt-3 col-md-12">
+                    <input
+                        type="radio"
+                        id="radioNoFakeMeeting"
+                        data-ng-model="$ctrl.select.meetingSlots.fakeMeeting"
+                        data-ng-value="false"
+                    />
+                    <label
+                        for="radioNoFakeMeeting"
+                        class="pl-2"
+                        data-translate="xdsl_meeting_no_fake_meeting"
+                    >
+                    </label>
+                </div>
+            </div>
+            <div
```

Comment:
**suggestion:** Same comment as @varun257, could you please use a `<oui-message>` element?